Bikeini/screens/Profile.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import {
+  StyleSheet, Text, View, ScrollView, Image, FlatList, TouchableOpacity, TouchableHighlight,
+} from 'react-native';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import headerStyle from './header';
+import serverApi from '../utilities/serverApi';
+import Item from '../components/Item';
+
+const profilePic = require('../assets/images/biker.png');
+
+
+const styles = StyleSheet.create({
+  background: {
+    backgroundColor: '#fff',
+  },
+  container: {
+    flex: 1,
+    alignItems: 'flex-start',
+    justifyContent: 'flex-start',
+  },
+  profile: {
+    height: 100,
+    width: 100,
+    margin: 10,
+  },
+  rowContainer: {
+    flexDirection: 'row',
+    backgroundColor: '#fff',
+  },
+  columnContainer: {
+    flexDirection: 'column',
+    backgroundColor: '#fff',
+    margin: 20,
+  },
+  categories: {
+    fontSize: 24,
+    margin: 10,
+  },
+  UserInfo: {
+    fontSize: 18,
+  },
+  greenButton: {
+    backgroundColor: '#44ccad',
+  },
+  greenButtonText: {
+    color: 'white',
+  },
+  editButtonContainer: {
+    height: 32,
+    flexDirection: 'row',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginBottom: 20,
+    width: 100,
+    borderRadius: 5,
+  },
+  actionButton: {
+    backgroundColor: '#00b5ec',
+  },
+});
+
+class Profile extends React.Component {
+    static navigationOptions = {
+      ...headerStyle,
+    };
+
+    constructor(props) {
+      super(props);
+      this.state = {
+        yourBicycles: '',
+      };
+    }
+
+    componentDidMount() {
+      this.getItemFromServer();
+    }
+
+    getItemFromServer = () => {
+      const { authState } = this.props;
+      const { jwt } = authState;
+
+      const yourBicycles = [];
+
+      serverApi.get('bikes/getmybikes/', jwt[0])
+        .then((responseJson) => {
+          for (let i = 0; i < responseJson.length; i += 1) {
+            yourBicycles.push(responseJson[i]);
+          }
```

Comment:
this is a shared function and should soner or later be moved to either action to a shared import